lib/index.js

```diff
@@ -9,14 +9,14 @@ const packageJsonFile = editJsonFile('./package.json', { autosave: true });
 const devDependencies = packageJsonFile.get('devDependencies');
 const dependencies = packageJsonFile.get('dependencies');
 
-const hasChanges = false;
+let hasChanges = false;
```

Comment:
it couses
```bash
TypeError: Assignment to constant variable.
    at setDependency (/Users/coffeeich/.nvm/versions/node/v8.6.0/lib/node_modules/fix-package-versions/lib/index.js:31:16)
    at Object.keys.forEach.dependency (/Users/coffeeich/.nvm/versions/node/v8.6.0/lib/node_modules/fix-package-versions/lib/index.js:21:7)
    at Array.forEach (<anonymous>)
    at fixVersion (/Users/coffeeich/.nvm/versions/node/v8.6.0/lib/node_modules/fix-package-versions/lib/index.js:18:29)
    at Object.fixAll (/Users/coffeeich/.nvm/versions/node/v8.6.0/lib/node_modules/fix-package-versions/lib/index.js:44:5)
    at repl:1:113
    at ContextifyScript.Script.runInThisContext (vm.js:50:33)
    at REPLServer.defaultEval (repl.js:239:29)
    at bound (domain.js:301:14)
    at REPLServer.runBound [as eval] (domain.js:314:12)
>
```

Reply:
I'm glad to listen that lib can be useful. You're correct, ill accept the PR.

Hope you will be safe for versioning problems in the future.

Reply:
Thx a lot!